### ISSUE TITLE
Can not convert model lm_model4_opt.onnx to tflite

### ISSUE THREAD

**Issue by tanphan07** [emilianavt/OpenSeeFace#69]
Thank you for your awesome project. 
I have a problem when converting model lm_model4_opt.onnx to tflite.
I use this repo https://github.com/onnx/onnx-tensorflow to convert onnx to tflite and get this error **BackendIsNotSupposedToImplementIt: FusedConv is not implemented**. Can you support me to handle this case.
Thank you very much

**Comment by emilianavt** [comment]
You may need to convert the original pytorch models in #1, because the FusedConv operation is not supported by some frameworks.

**Comment by tanphan07** [comment]
Thank for your reply, but i can not found the lm_model4 in the https://github.com/emilianavt/OpenSeeFace/issues/48. Moreover, can you give me an advice training a model that has a good performance when eyes closed

**Comment by emilianavt** [comment]
>Thank for your reply, but i can not find the lm_model4 in the https://github.com/emilianavt/OpenSeeFace/issues/48.
In that case, you may have to reconvert it from pytorch weights, which can maybe, hopefully be found in #1.

>Moreover, can you give me an advice training a model that has a good performance when eyes closed
Make sure that you have a lot of correctly annotated images in various head poses with the eyes closed.

**Comment by tanphan07** [comment]
> > Thank for your reply, but i can not find the lm_model4 in the #48.
> 
> In that case, you may have to reconvert it from pytorch weights, which can maybe, hopefully be found in #1. You can also try using model 3. It should have roughly the same tracking quality. 4 just has better support for winks (a single closed eye).
> 
> > Moreover, can you give me an advice training a model that has a good performance when eyes closed
> 
> Make sure that you have a lot of correctly annotated images in various head poses with the eyes closed.

In my use case, i want a model that can make high accuracy when the eyes closed, so i think only model 4 that can help me. Can you provide checkpoint of model 4. Thank you very much.

**Comment by emilianavt** [comment]
The accuracy of model 4 shouldn't be higher when both eyes are closed. It is only better at decoupling the closedness state of the eyes.

It may take some time before I can find the original checkpoint.

**Comment by tanphan07** [comment]
> The accuracy of model 4 shouldn't be higher when both eyes are closed. It is only better at decoupling the closedness state of the eyes.
> 
> It may take some time before I can find the original checkpoint.

could you provide the dataset that has a lot of correctly annotated images in various head poses with the eyes closed? Thank you in advance

**Comment by emilianavt** [comment]
No, I cannot.
